go/vt/wrangler/materializer_test.go

```diff
@@ -1162,6 +1162,126 @@ func TestCreateLookupVindexSameKeyspace(t *testing.T) {
 		t.Errorf("same keyspace: got:\n%v, want\n%v", got, want)
 	}
 }
+
+func TestCreateLookupVindexCustomHashing(t *testing.T) {
+	ms := &vtctldatapb.MaterializeSettings{
+		SourceKeyspace: "ks",
+		TargetKeyspace: "ks",
+	}
+	env := newTestMaterializerEnv(t, ms, []string{"0"}, []string{"0"})
+	defer env.close()
+
+	specs := &vschemapb.Keyspace{
```

Comment:
Nit, but this would be easier to read/follow if the lookup vindex spec followed the schema and vschema definitions.